public/main.ts

```diff
@@ -56,6 +65,11 @@ class Farm {
         }
         console.assert(this.stockpile > 0);
     }
+    playerHarvest(crop: Crop) {
+        // TODO: add upgrades to click rate
+        this.stockpile += clickGrowthRate * (1 + this.totalFarmers);
+        this.stockpile = Math.ceil(this.stockpile);
```

Comment:
Could be merged into one line.
```suggestion
        this.stockpile += Math.ceil(clickGrowthRate * (1 + this.totalFarmers));
```

I am trying to get the two assignments down to one.